frontend/src/app/coworking/widgets/room-reservation-table/room-reservation-table.widget.ts

```diff
@@ -148,9 +182,14 @@ export class RoomReservationWidgetComponent {
       },
 
       (error) => {
-        // Handle errors here
-        console.error('Error drafting reservation', error);
-        this.snackBar.open(error.error.message, 'Close', this.snackBarOptions);
+        let errorMessage = 'Unknown error';
+        try {
+          const errorObj = JSON.parse(error.message);
+          errorMessage = errorObj.error.message;
```

Comment:
Nit: Is it possible just to condense this?

Reply:
Looping in @aarjavjain2002